addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -24,13 +24,21 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
 
   constructor() {
     super(...arguments);
+
     this.loadOptions();
-    this.loadProvidedValue();
+    // this.loadProvidedValue();
   }
 
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
-    const fieldOptions = this.args.field.options;
+    const fieldOptions = this.getFieldOptionsAsObject(this.args.field.options);
```

Comment:
Strange, `field.options` should already be an object https://github.com/lblod/ember-submission-form-fields/blob/7cee9c3f63eab1167da6a0c19fb41727a8000f17/addon/models/field.js#L94-L104

Are there scenarios where it isn't?

Reply:
🤦🏻  I was not aware of this..

In the form builder I keep getting this issue that `value` is not accessable. not sure if you saw the ticket? but this was the error 
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/3d1cf553-297e-4886-bcc8-3b3f35c8be5e)

I should remove the `toObject` part and than we  should check how relevant the `validation` is on the options is.

Reply:
I haven't looked at Jira yet, no. Just catching up on emails (Github notifications) / rocket chat at the moment 😄.